cli/cmd/encore/init_windows.go

```diff
@@ -0,0 +1,21 @@
+// +build windows
+
+package main
+
+import (
+	"golang.org/x/sys/windows"
+)
+
+// init activates virtual terminal feature on "windows", this enables colored
+// terminal output.
+func init() {
+	setConsoleMode(windows.Stdout, windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING)
+	setConsoleMode(windows.Stderr, windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING)
+}
+
+// setConsoleMode enables VT processing on stout and stderr.
+func setConsoleMode(handle windows.Handle, flag uint32) {
+	var mode uint32
+	windows.GetConsoleMode(handle, &mode)
```

Comment:
If GetConsoleMode fails it'll leave mode as zero. If then SetConsoleMode succeeds we'll set the wrong mode. We should guard the SetConsoleMode behind an err check.

Reply:
Totally agree!